src/app/error/error.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+    <img src="../assets/goal.jpg" alt="goal">
```

Comment:
There is an image already for 5xx errors - src/app/error-handling/img/injury.jpg